staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -1181,6 +1186,310 @@ func RunTestList(ctx context.Context, t *testing.T, store storage.Interface, com
 			expectRVFunc: resourceVersionNotOlderThan(list.ResourceVersion),
 			expectedOut:  []example.Pod{},
 		},
+		// match=Exact
```

Comment:
nit: can you extract the new test cases to a separate PR (so that if we will have to rollback or sth, we will not have to rollback the refactoring part)?